frontend/src/app/dora/components/config-stepper/reports/reports.component.spec.ts

```diff
@@ -1,26 +1,91 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { ExportComponent } from './reports.component';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { SimpleChange } from '@angular/core';
+import { ApiService } from 'src/app/dora/service/api.service';
+import { ReportParams } from '../../../models/reportParams';
+import { BoardParams } from 'src/app/dora/models/boardParams';
+import { metrics } from 'src/app/dora/utils/config';
+import { of } from 'rxjs';
+import { ReportResponse } from '../../../types/reportResponse';
 
 describe('ExportComponent', () => {
-  let component: ExportComponent;
+  let exportComponent: ExportComponent;
   let fixture: ComponentFixture<ExportComponent>;
+  let element: any;
+  let apiService: ApiService;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [HttpClientTestingModule],
       declarations: [ExportComponent],
+      providers: [
+        {
+          useValue: { metrics: metrics },
+        },
+      ],
     }).compileComponents();
   }));
 
   beforeEach(() => {
     fixture = TestBed.createComponent(ExportComponent);
-    component = fixture.componentInstance;
+    apiService = TestBed.inject(ApiService);
+    exportComponent = fixture.componentInstance;
+    element = fixture.nativeElement;
     fixture.detectChanges();
   });
 
   it('should create', () => {
-    expect(component).toBeTruthy();
+    expect(exportComponent).toBeTruthy();
+  });
+
+  it('should call ngOnChanges', () => {
+    const kanbanSetting = new BoardParams({
+      type: 'jira',
+      token: 'test-token',
+      site: 'dorametrics',
```

Comment:
maybe we can use a fake site in there replaces the truthfully data `dorametrics`